src/VisualStudio/Core/Def/Implementation/Snippets/SnippetFunctions/AbstractSnippetFunctionArgumentValue.cs

```diff
@@ -0,0 +1,74 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Threading;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Completion;
+using Microsoft.CodeAnalysis.Shared.Extensions;
+using Microsoft.CodeAnalysis.Text;
+using Microsoft.VisualStudio.Text;
+using Roslyn.Utilities;
+using VsTextSpan = Microsoft.VisualStudio.TextManager.Interop.TextSpan;
+
+namespace Microsoft.VisualStudio.LanguageServices.Implementation.Snippets
+{
+    internal abstract class AbstractSnippetFunctionArgumentValue : AbstractSnippetFunction
+    {
+        private readonly string _parameterName;
+        private readonly SymbolKey _parameter;
+
+        protected AbstractSnippetFunctionArgumentValue(AbstractSnippetExpansionClient snippetExpansionClient, ITextBuffer subjectBuffer, string fieldName, string parameter)
+            : base(snippetExpansionClient, subjectBuffer)
+        {
+            _parameterName = fieldName;
```

Comment:
mismatch between names.

Reply:
➡️ Fixing as part of https://github.com/dotnet/roslyn/pull/49916#discussion_r544814275